src/client/js/components/Admin/SlackIntegration/SlackIntegration.jsx

```diff
@@ -143,16 +143,18 @@ const SlackIntegration = (props) => {
 
         {t('admin:slack_integration.selecting_bot_types.selecting_bot_type')}
 
-        <div className="row my-4">
-          <div className="card-deck mx-auto">
+        <div className="my-4">
+          <div className="row flex-wrap-reverse justify-content-center">
```

Comment:
ここ分ける必要があんまりなさそうですね
```suggestion
          <div className="row my-4 flex-wrap-reverse justify-content-center">
```
で良さそう？